docker/README.md

```diff
@@ -74,6 +74,12 @@ You can set the following variables on a per-command basis (`PORT_WORDPRESS=8000
 Configurable settings are documented in the [`./docker/default.env` file](https://github.com/Automattic/jetpack/blob/master/docker/default.env).
 Customizations should go into a `./docker/.env` file you create, though, not in the `./docker/default.env` file.
 
+### Mounting extra directories into the container
```

Comment:
There's a section "Custom plugins & themes in the container" which does basically the same. Let's combine those into one for clarity 💪

Reply:
It's already done in bdf54dfe849fc940823c02e2522381754f6142e5. Content was combined. The difference is that there was no providead means of letting the extra config file live with untracked modifications inside the jetpack directory. Now, `yarn docker:up` will create the file from a template and load it for you without having to run the command with extra options in the command line

Reply:
Basically this branchI should have pushed it years ago when we were discusing the extra `-f file.yml` file thing over DM when the instructions were laid out in the README.